### PR TITLE
Bug 1786068: Bug 1786070: Bug 1786091: Fixes kubevirt cdrom bugs

### PR DESCRIPTION
**Pull request by glekner**
- CD Deletion now happens only after submitting
- Removed Eject mechanism
- Set Hover delay of max cd tooltip to 0

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1786068
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1786070
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1786091

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/k8s/patches/vm/vm-cdrom-patches.ts [hunk 1]
```diff
@@ -40,112 +40,113 @@ const assignBootOrderIndex = (vm: VMLikeEntityKind, currDevBootOrder = -1) => {
...
-  let DISKS = getDisks(asVM(vm));
+
+  let DISKS = getDisks(asVM(vm)).filter(
+    (disk) => !disk.cdrom || cds.find((modalCD) => disk.name === modalCD.name || modalCD.newCD),
+  );
   let VOLS = getVolumes(asVM(vm));
   let DATATEMPLATES = getDataVolumeTemplates(asVM(vm));
   const storageClassConfigMap = await getStorageClassConfigMap({ k8sGet });
 
-  cds
```
Comment (glekner) [inline frontend/packages/kubevirt-plugin/src/k8s/patches/vm/vm-cdrom-patches.ts]: For Reviewers
The only change here is removal of `cds.filter((cd) => cd.changed)`

**Comment by yaacov** [comment]
/lgtm
+1 on the red/green ratio

**Pull request merged**